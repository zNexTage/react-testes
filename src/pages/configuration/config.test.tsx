import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Configuration from "./configuration";


jest.mock('react-router-dom');

describe("A configuration page", ()=>{
    test("Should be rendered", ()=>{
        const {container} = render(
            <RecoilRoot>
                <Configuration />
            </RecoilRoot>
        )

        expect(container).toMatchSnapshot();
    });
});