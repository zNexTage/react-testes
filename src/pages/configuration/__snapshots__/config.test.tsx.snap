// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A configuration page Should be rendered 1`] = `
<div>
  <header
    class="header"
  >
    <picture
      class="header__logo"
    >
      <source
        media="(max-width: 790px)"
        srcset="logo-pequeno.png"
      />
      <img
        alt=""
        class="header__logo"
        src="logo.png"
      />
    </picture>
    <img
      alt=""
      class="header__participant"
      src="participante.png"
    />
  </header>
  <main
    class="bg-blue h-100"
  >
    <section
      class="bg-beige h-100 configuration__main__section"
    >
      <form>
        <fieldset
          class="form__fieldset"
        >
          <legend
            class="form__legend"
          >
            <h1>
              Vamos come√ßar?
            </h1>
          </legend>
          <div>
            <input
              class="form__input"
              placeholder="Insira os nomes dos participantes"
              type="text"
              value=""
            />
            <button
              class="form__button"
              disabled=""
            >
              Adicionar
            </button>
          </div>
        </fieldset>
      </form>
      <ul />
      <footer>
        <button
          disabled=""
        >
          Iniciar brincadeira
        </button>
      </footer>
    </section>
  </main>
</div>
`;
